<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>viewTestimonials</class>
 <widget class="QDialog" name="viewTestimonials">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>738</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextBrowser" name="priorTestimonials">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>971</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="addReviewLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Your Review:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="userTestimonial">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>971</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addTestimonial">
   <property name="geometry">
    <rect>
     <x>859</x>
     <y>690</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Review</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>290</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QHBoxLayout" name="getName">
      <item>
       <widget class="QLabel" name="name">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="username"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="getRating">
      <item>
       <widget class="QLineEdit" name="rating"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>/5</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
